rc/base/screen.kak

```diff
@@ -0,0 +1,48 @@
+# http://gnu.org/software/screen/
+# ‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾
+ 
+hook global KakBegin .* %{
+    %sh{
+        [ -z "$STY"] && exit
+        echo "
+            alias global focus screen-focus
+            alias global new screen-new-vertical
+        "
+    }
+}
+
+
+def screen-new-vertical -params .. -command-completion -docstring "Create a new vertical region" %{
+     %sh{
+       cmd="screen -X eval 'split -h' 'focus down' 'screen kak -c ${kak_session} -e \"$@\"'"
+       eval ${cmd} 2>&1 >/dev/null
```

Comment:
If `cmd` has to stay, then this expansion needs to be enclosed in quotes. Also not hiding errors may be better, to track issues.